### PR TITLE
[NL] Add 25 new translations

### PR DESCRIPTION
**Pull request by Joppe27**
Dutch translations of 25 (shorter) wiki articles.

Little bit of help needed for Brand_identity_guidelines and Gameplay___Drain_time, info in commit comments.

Formatting errors due to incompatibilties with my text editing software can occur, please point them out!

### CODE REVIEW

**Comment by TPGPL** [comment]
Thanks for your contributions! Although next time, try to make one PR for one translation, as it's much harder to review and keep files up-to-date in a PR with several translations

**Comment by Joppe27** [comment]
Sorry, I thought that would be too much spam. I'll post them seperately next time

**Comment by Joppe27** [comment]
Do I actually need to edit all of the articles until that remark bot stops failing checks or are those just suggestions? It's giving me an error on each and every line and I don't understand what it wants me to change. I write on Windows software, I can't use the unix format for linebreaks (I don't even know what those are).

**Comment by MegaApplePi** [comment]
@Joppe27 Generally yes, you would go ahead and apply those changes to make remark happy (although note that some others are still making some adjustments to it). It looks like you used GitHub to create those files and it decided to use the Windows style line ending for those files.

I can apply the fix for this part of the failed check.

**Comment by Joppe27** [comment]
> 
> 
> @Joppe27 Generally yes, you would go ahead and apply those changes to make remark happy (although note that some others are still making some adjustments to it). It looks like you used GitHub to create those files and it decided to use the Windows style line ending for those files.
> 
> I can apply the fix for this part of the failed check.

Thanks! Could you tell me how you converted the line endings? I can't find an option to change them in my writing program so I'll probably have to convert every time I upload a new article

File: wiki/Project_Loved/nl.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Project Loved
```
Comment (TPGPL) [inline wiki/Project_Loved/nl.md:1]: outdated translation

File: wiki/BanchoBot/nl.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
+# BanchoBot
```
Comment (TPGPL) [inline wiki/BanchoBot/nl.md:1]: translate image captions and alt-texts

File: wiki/Beatmap_Discussion/nl.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+De auter(s) van een post op een discussiepagina kunnen formatteringshulpmiddelen gebruiken om het gebruiksgemak van de post te verbeteren. Dit omvat permalinks, aanpassingen en verwijderingen.
+
+`Permalink` neemt een directe link naar een post. Dit zal getoond worden als een referentienummer (bv. `#1234567`) aan andere gebruikers, deze kan aangeklikt worden om snel naar de desbetreffende post te navigeren. Dit is vooral nuttig bij het verwijzen naar problemen over verschillende modposts.
+
+`Edit` geeft gebruikers de mogelijkheid om hun inzeding aan te passen in het geval van fouten. Het is niet de bedoeling dat deze functie gebruikt wordt om op berichten te antwoorden. Maak in plaats daarvan een nieuw antwoord aan.
+
+`Delete` staat gebruikers toe om hun inzending te verwijderen als er serieuze fouten in staan. Deze functie is uitgeschakeld voor posts met veel discussie om het verwijderen van posts door verhitte discussie te voorkomen.
+
+<!--TODO: insert lots of links-->
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/nl.md:162]: remove comment

File: wiki/BanchoBot/nl.md (hunk 2 above)
Comment (Joppe27) [reply wiki/BanchoBot/nl.md:1]: What do you mean by alt-texts?

File: wiki/Twitter/nl.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+osu! heeft verschillende Twitter-accounts, die elk een specifiek doel hebben. De meeste van deze accounts tweeten niet zo vaak, maar ze helpen je om geïnformeerd te blijven buiten de osu! omgeving.
+
+| Avatar | Gebruikersnaam | Beschrijving |
+| :-: | :-: | :-- |
+| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | De officiële bron van nieuws en aankondigingen. Weinig tweets, hoge kwaliteit. |
+| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Meldingen over site- en banchoproblemen vind je hier! |
+| ![Dean Herbert avatar](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Niet altijd osu!-gerelateerd, maar we zullen dit hier laten voor de geïnteresseerden. ppy’s leven is tenslotte grotendeels osu! |
+| ![osu! support avatar](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Voor hulp met account- of communityproblemen. |
+| ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Komische tweets wanneer er dingen fout lopen. |
```
[the remark this answers is not in the document]
Comment (Joppe27) [reply wiki/Twitter/nl.md:13]: Avatar is a loan word in dutch, editing not needed

File: wiki/Project_Loved/nl.md (hunk 1 above)
Comment (Joppe27) [reply wiki/Project_Loved/nl.md:1]: This one is the same as master, I can't find the new version?

Comment (TPGPL) [reply wiki/Project_Loved/nl.md:1]: your translation has two `Meer informatie` headers and 6 headers in general, while the English version has 5 headers

File: wiki/Beatmap_Discussion/nl.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
+---
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/nl.md:1]: image captions not translated

File: wiki/Do_you_really_want_to_ask_peppy/nl.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<!-- TODO: shouldn't be written from peppy's perspective. it could probably be converted into a general "who should you contact" page -->
```
Comment (TPGPL) [inline wiki/Do_you_really_want_to_ask_peppy/nl.md:1]: original article has been revised recently, translation needs updating

File: wiki/Featured_Artists/nl.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+---
+tags:
+- FA
+---
+
+# Aanbevolen artiesten
+
+**Aanbevolen artiesten (Featured Artists)** zijn muziekartiesten die met osu! samenwerkten om een selectie van hun nummers beschikbaar te maken voor het gebruik in [beatmaps](/wiki/Beatmaps) en als achtergrondmuziek in osu!'s World Cup <!-- TODO: "World Cup" should be able to link somewhere --> livestreams.
```
Comment (TPGPL) [inline wiki/Featured_Artists/nl.md:8]: remove comment

File: wiki/Featured_Artists/nl.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+Alle beatmappingwedstrijden die op de [communitywedstrijdpagina](https://osu.ppy.sh/community/contests) gepubliceerd werden, zoals de [Maandelijke Beatmappingwedstrijd](/wiki/Contests/Monthly_Beatmapping_Contest) en de [Mappingolympiade](/wiki/Contests#osu!-mapping-olympiad), vereisen dat deelnemers een nummer van een aanbevolen artiest beatmappen. Sommige door de community georganiseerde wedstrijden gebruiken deze nummers ook.
+
+Om het beatmappen van aanbevolen artiest nummers te promoten hebben leden van de community de [Mappers' Gilde](/wiki/Mappers_Guild) opgericht, die beatmappers een quest-systeem in MMO-stijl geeft waaraan ze samen met andere mappers kunnen deelnemen. Het maken en ranken van beatmaps wordt beloond met prijzen, waaronder drie niveaus van profielbadges<!-- TODO: profile badges article -->. Beatmappers kunnen zich aansluiten bij de gilde door de [Mappers' Gilde-website](https://mappersguild.com/) te bezoeken.
```
Comment (TPGPL) [inline wiki/Featured_Artists/nl.md:38]: remove comments

File: wiki/Project_Loved/nl.md (hunk 1 above)
Comment (Joppe27) [reply wiki/Project_Loved/nl.md:1]: Sorry, no idea what happened there. Fixed!

File: wiki/Beatmap_Discussion/nl.md (hunk 5 above)
Comment (TPGPL) [reply wiki/Beatmap_Discussion/nl.md:1]: there are still some untranslated ones:
L106
L120
L130

Comment (Joppe27) [reply wiki/Beatmap_Discussion/nl.md:1]: Sorry, ctrl+f sucks. Will add in a moment.

**Pull request merged**